nixfmt.cabal

```diff
@@ -6,10 +6,20 @@ build-type:          Simple
 
 executable nixfmt
```

Comment:
It is not a good idea to put everything into the executable. You usually want to keep the amount of code in the executable as small as possible and put most of the functionality into the library, so that it can be reused to others, or by yourself later, or so that you can write tests.

Reply:
(I would also recommend using `package.yaml` instead of a cabal file.)